backend/src/test/java/ai/verta/modeldb/FindProjectEntitiesTest.java

```diff
@@ -272,7 +274,7 @@ private static void createProjectEntities() {
             .addTags("Tag_5")
             .addTags("Tag_7")
             .addTags("Tag_8")
-            .setProjectVisibility(ProjectVisibility.PUBLIC)
+            .setVisibility(ResourceVisibility.PRIVATE)
```

Comment:
This is going from public to private. I don't know if it breaks anything.

Reply:
For this test is should not matter, since the visibility is not part of the criteria under test.  I'll double check it to be sure.